eclipse-collections-code-generator/src/main/resources/api/map/primitivePrimitiveMap.stg

```diff
@@ -81,6 +82,21 @@ public interface <name1><name2>Map extends <name2>ValuesMap
      */
     void forEachKeyValue(<name1><name2>Procedure procedure);
 
+    /**
+     * Implements the injectInto pattern with each key/value pair of the map.
+     * @param value to be injected into the map
+     * @param function to apply to the injected value and key/value pairs
+     * @return result of injecting the injectedValue into the map
+     * @since 11.1
+     */
+    default \<IV> IV injectIntoKeyValue(IV injectedValue,
+    Object<name1><name2>ToObjectFunction\<? super IV, ? extends IV> function)
+    {
```

Comment:
Move the `Object<name1><name2>ToObjectFunction\<? super IV, ? extends IV> function)` to the previous line as it is failing off a checkstyle indentation rule.